packages/rest/src/rest.component.ts

```diff
@@ -52,6 +52,4 @@ export class RestComponent implements Component {
   }
 }
 
-export interface RestComponentConfig extends RestServerConfig {
-  // TODO(kevin): Extend this interface def to include multiple servers?
```

Comment:
Please preserve this comment, e.g. on the line preceding `export type RestComponentConfig`.